lib/routes/account.js

```diff
@@ -1095,41 +1077,40 @@ module.exports = function (
         var sessionToken = request.auth.credentials
         var service = request.payload.service || request.query.service
 
-        return db.sessionTokenWithVerificationStatus(sessionToken.id)
-          .then(function (token) {
+        var doSigninConfirmation = requestHelper.shouldEnableSigninConfirmation(sessionToken, config, request)
```

Comment:
Do we need to call `shouldEnableSigninConfirmation` here?

It seems to me that we can just check the existence of `sessionToken.tokenVerificationId`. If this user wasn't part of the cohort, `tokenVerificationId` wouldn't have been specified at the point of creation. Right? Or wrong?